pkg/providers/vsphere/vmprovider_vm_resize_test.go

```diff
@@ -222,6 +232,36 @@ func vmResizeTests() {
 			})
 		})
 
+		Context("Powered On VM", func() {
+			BeforeEach(func() {
+				configSpec.NumCPUs = 2
+				configSpec.MemoryMB = 1024
+			})
+
+			It("Resize Pending", func() {
```

Comment:
Can we also add a `Resize Succeed` test?

Reply:
This test was cribbed from the immediate prior "Powering On VM" resize test. For this test a powered on VM won't be resized - the primary purpose of this annotation is to indicate that a resize is avail and would be done the next time the VM is powered off.